tests/linalg_test.py

```diff
@@ -2357,5 +2358,59 @@ def testPascal(self, n, kind):
     self._CompileAndCheck(jsp_fun, args_maker)
 
 
+class TestSolveSylvester(jtu.JaxTestCase):
+  # float32 is not close enough for assert
+  jax.config.update("jax_enable_x64", True)  # Ensure float64 for higher precision
```

Comment:
Please do not update configurations as part of the test. Every test file is run in both 32 and 64-bit precision; if 32-bit precision is problematic for some reason, we can skip those tests (and note this in the function documentation)